diff --git a/out/LinuxUpdater.js b/out/LinuxUpdater.js
index a7587ae..7cb3011 100644
--- a/out/LinuxUpdater.js
+++ b/out/LinuxUpdater.js
@@ -34,7 +34,7 @@ class LinuxUpdater extends BaseUpdater_1.BaseUpdater {
         if (/pkexec/i.test(sudo[0]) || sudo[0] === "sudo") {
             wrapper = "";
         }
-        return this.spawnSyncLog(sudo[0], [...(sudo.length > 1 ? sudo.slice(1) : []), `${wrapper}/bin/bash`, "-c", `'${commandWithArgs.join(" ")}'${wrapper}`]);
+        return this.spawnSyncLog(sudo[0], [...(sudo.length > 1 ? sudo.slice(1) : []), `${wrapper}/bin/bash`, "-c", wrapper === "" ? commandWithArgs.join(" ") : `'${commandWithArgs.join(" ")}'${wrapper}`]);
     }
     sudoWithArgs(installComment) {
         const sudo = this.determineSudoCommand();
